src/lib.rs

```diff
@@ -56,31 +54,32 @@ mod env;
 mod error;
 mod log_util;
 mod metadata;
+mod not_sync;
```

Comment:
So this rename is to avoid clashing with a potential keyword?

Reply:
Ah it's mentioned in the OP.
Resolving.

Reply:
@ice1000 Yes, `async` is a keyword in 2018 that otherwise needs to be written as `r#async` (raw identifier). @nrc above suggested to just use `r#async`, which is also reasonable (and the word "async" is certainly more sensible than "not_sync".

Do you have opinions about what this module should be called? Whether it should stay `r#async`?